emcc.py

```diff
@@ -1276,6 +1277,8 @@ def consume_arg_file():
       options.source_map_base = consume_arg()
     elif check_arg('--embind-emit-tsd'):
       options.embind_emit_tsd = consume_arg()
+    elif check_arg('--emit-tsd'):
```

Comment:
In that case it might be worth marking it as experimental by doing something like we do for wasm64: `diagnostics.warning('experimental', '-sMEMORY64 is still experimental. Many features may not work.')`